.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,15 @@
-## Purpose of this pull request
+## PLEASE READ
+
+Following a recent back-end update, all contributors with a _local clone of our repository_ are required to run `yarn install`. This does not apply to forked pull requests.
```

Comment:
@kimsauce, it says, "This does not apply to forked pull requests." But we've had problems with forked repos too. Are you sure they don't need to run `yarn install` in forked repos?

Reply:
You're correct! It does apply to those with physical, local forks of our repo, but doesn't apply to those who follow the ["Edit this page" workflow](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits), as that doesn't involve cloning/forking locally. Can you take a look at this revised wording when you can?

Reply:
Looks good.